worker/uniter/remotestate/watcher.go

```diff
@@ -300,6 +299,12 @@ func (w *RemoteStateWatcher) loop(unitTag names.UnitTag) (err error) {
 		observedEvent(&seenLeadershipChange)
 	}
 
+	// TODO(wallyworld) - listen for changes to this value
```

Comment:
What is the plan around this? update-status interval is currently fixed and you can change the setting and restart Jujud?